SocialFilter/sections/specific.txt

```diff
@@ -88,6 +88,7 @@ tvp.info##.nav__fake--social
 zapytaj.onet.pl###fbFixed
 mjakmama24.pl##body > #panel[style="left:0; position:fixed; top:30%; z-index:6000;"]
 app-time.ru##.wrap-social
+f1-gate.com##.content > .contentbody > .sns
```

Comment:
`f1-gate.com##.sns` is acceptable in this case(take a look on the similar rules in social filter)